### PR TITLE
README, html and css files from pre-trials

### PR DESCRIPTION
**Pull request by annaeliana7**
In this feature I: 

- Added an html, css and README file
- Copied and pasted html and css files from pre-trials repository
- Added and updated a README file

### CODE REVIEW

**Comment by annaeliana7** [comment]
Your project is almost complete... Great job :raised_hands: However, some slight changes are required

TO HIGHLIGHT: :tada:

- Your project is professional
- Your repo is professional
- Well structured files

**Review by aimalamiri — CHANGES_REQUESTED** [review]
Hi @annaeliana7, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

AlmostThereReachGIF

## Highlights

- The comment is professional ✔️ ⭐
- Good job for using the right GitHub review comments 💯

## Required Changes ♻️

- [ ] **You did a great job, but it's required to ask for changes, and also add at least two comments, and use the [GitHub inline comments feature](https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request). Because this is the opportunity for you to practice performing a code review in a professional way. So please make sure to add another review in this pull request.**


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  <title>Fork Me! FCC: Test Suite Template</title>
+  <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+  <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+  <link rel="stylesheet" href="styles.css">
+  <main id="main">
+    <h1 id="title">Amelia Earhart
```
Comment (annaeliana7) [inline index.html:15]: Kindly close the h1 tag in your html file

File: index.html [hunk 2]
```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <title>Fork Me! FCC: Test Suite Template</title>
+  <link rel="stylesheet" href="css/style.css">
```
Comment (annaeliana7) [inline index.html:7]: Your style.css is not linked to your html. Therefore your project is previewing html only. Kindly attach your css to your html file and ensure that the link works

File: style.css [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    text-align:center;
+    color:white;
+  }
+  #image{
+    max-width:none;
+    height:auto;
+    display:block;
+    margin-left:auto;
+    margin-right:auto;
+  }
```
Comment (annaeliana7) [inline style.css:14]: The element should responsively resize, relative to the width of its parent element, without exceeding its original size. Kindly adjust the max-width. Thank you.

**Review by annaeliana7 — COMMENTED** [review]
Your project is almost complete... Great job 🙌 However, some slight changes are required

TO HIGHLIGHT: 🎉

- Your project is professional
- Your repo is professional
- Well structured files

### **Status: REQUIRED CHANGES ♻️** 
_Check the comments under the review._


### Optional suggestions

- N/A

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.
Cheers and Happy coding!:clap::clap::clap:
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**Review by GiuseppeTG — APPROVED** [review]
Hi @annaeliana7  👋 , 

Your project is complete!  Very good Job 👍 

Alt Text

## Highlights
✅ Good use of inline comments
✅ Professional review

## Optional suggestions

It is good to use some emojis in your review, they make the code review more human-friendly 👽🤖.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**